plugins/statistics.koplugin/main.lua

```diff
@@ -177,14 +178,25 @@ function ReaderStatistics:addToMainMenu(tab_item_table)
                 end
             },
             {
+                text = _("Reading progress"),
+                callback = function()
+                    UIManager:setDirty(nil, "full")
```

Comment:
Is this really required? If it's not required for other menu items, then we should figure out why it's not working for the progress widget. What error were you getting?